.github/workflows/generator_container_slsa3.yml

```diff
@@ -70,6 +76,14 @@ on:
         required: false
         type: string
         default: ""
+      provenance-registry-username:
+          description: "Username when publishing to provenance registry instead of image registry"
+          required: false
+          type: string
+      provenance-registry:
```

Comment:
Can we remove this from secrets? There's already one in the `inputs` section, correct? Users can use `VAR` to avoid having the registry name hardcoded in their repo https://docs.github.com/en/actions/learn-github-actions/variables

Reply:
removed `provenance-registry` secret